handler.go

```diff
@@ -105,4 +105,34 @@ func xmlResponse(w *Response, r *Request) {
 	w.Body = expectedBody
 }
 
+func kittehResponse(w *Response, r *Request) {
+	applyStatusToResponse(w, StatusOK)
+	w.Header.Del("Content-Type")
+	w.Header.Add("Content-Type", "image/jpeg")
+	expectedBody := "test body"
+	w.Body = expectedBody
+}
+
+func doggoResponse(w *Response, r *Request) {
+	applyStatusToResponse(w, StatusOK)
+	w.Header.Del("Content-Type")
+	w.Header.Add("Content-Type", "image/png")
+	expectedBody := "test body"
+	w.Body = expectedBody
+}
 
+func kissesResponse(w *Response, r *Request) {
+	applyStatusToResponse(w, StatusOK)
+	w.Header.Del("Content-Type")
+	w.Header.Add("Content-Type", "image/gif")
+	expectedBody := "test body"
+	w.Body = expectedBody
+}
+
+func healthCheckResponse(w *Response, r *Request) {
+	applyStatusToResponse(w, StatusOK)
+	w.Header.Del("Content-Type")
```

Comment:
instead of having to delete the header and then adding you could refactor it so you replace existing when you `Header.Add`.

Reply:
✅